README.md

```diff
@@ -154,6 +140,48 @@ const suite = new EcdsaSepc256k1Signature2019({
 });
 ```
 
+### Custom documentLoader
+
+Pre-requisites:
+
+* You have an existing valid JSON-LD `@context`.
+* Your custom context is resolvable at an address.
+
+```js
+const vc = require('vc-js');
+// vc-js exports its own secure documentLoader.
+const {defaultDocumentLoader} = vc;
+// a valid json-ld @context.
+const myCustomContext = require('./myCustomContext');
+
+const documentLoader = async url => {
+  if(url === 'did:test:context:foo') {
+    return {
+      contextUrl: null,
+      documentUrl: url,
+      document: myCustomContext
+    };
+  }
+  return defaultDocumentLoader(url);
+};
```

Comment:
I think this may be exposing a deeper problem.  At a low level, jsigs is supposed to use its security contexts *first* (right?).  jsigs exposes `extendContextLoader` to help do that.  vc-js exposes the `defaultDocumentLoader` (used above) that extended the jsigs loader with a loader for some example contexts.  Now these docs are suggesting apps use a loader that does their checks first before calling out to the default loader.

So... if I'm following the stack correctly, this means the low level jsigs usage will be checking the *app loader first*, then fallback to the default loader which would check jsigs loader and then finally the vc-js loader.  If the intent is jsigs should use its local security contexts always first, I think we have to handle that internally in jsigs, and then just use regular override pattern going up the stack.  The jsigs extendContextLoader would be removed in that case I guess.

@dlongley Thoughts?

Reply:
I added `extendContextLoader` to the example along with a note:

https://github.com/digitalbazaar/vc-js/pull/33/commits/99982424c74b98d0012fcf9d1ca4b1869bf5b5c2

Reply:
@davidlehn I've opened an issue based on the concerns you brought up, #44.